sources/packages/backend/apps/api/src/route-controllers/restriction/restriction.controller.service.ts

```diff
@@ -0,0 +1,60 @@
+import { NotFoundException, Injectable } from "@nestjs/common";
+import { StudentRestrictionService } from "../../services";
+import { StudentRestriction } from "@sims/sims-db";
+
+@Injectable()
+export class RestrictionControllerService {
+  constructor(
+    private readonly studentRestrictionService: StudentRestrictionService,
+  ) {}
+
+  /**
+   * Get restrictions for a student.
+   * @param studentId id of the student to retrieve restrictions.
+   * @param options to filter restrictions.
+   * - `filterNoEffectRestrictions` option to filter restrictions based on notificationType.
+   * @returns student restrictions for the provided student id.
+   */
+  async getStudentRestrictions(
+    studentId: number,
+    options?: {
+      filterNoEffectRestrictions?: boolean;
```

Comment:
onlyActive is missing in the options. I am sure it has a consumer using the onlyActive option as true. Please make sure it's consumer is also modified accordingly.

Reply:
I believe that it is not used by any consumer. If true, can we remove it?